polaris.shopify.com/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect } from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+import GlobalSearch from "../GlobalSearch";
+import MaxPageWidthDiv from "../MaxPageWidthDiv";
+import Button from "../Button";
+import SideMenu from "../SideMenu";
+import NavItems from "../NavItems";
+
+import shopifyLogo from "../../../public/shopify-logo.svg";
+import hamburguerIcon from "../../../public/images/icon-hamburguer.svg";
+import styles from "./Header.module.scss";
+
+interface Props {
+  currentSection?: string;
+}
+
+function Header({ currentSection }: Props) {
+  const [showMenu, setShowMenu] = useState(false);
+
+  useEffect(() => {
+    // it ensures that the menu is closed after the screen is resized
+    function handleResize() {
+      if (window.innerWidth > 768 && showMenu) {
+        setShowMenu(false);
+      }
+    }
+
+    window.addEventListener("resize", handleResize);
+
+    return () => window.removeEventListener("resize", handleResize);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const handleCloseMenu = () => {
+    setShowMenu(false);
+    // it focus back to the menu button after the user closes the menu
+    document.getElementById("menu-button")?.focus();
```

Comment:
Could we use refs here instead? Seems like a more Reacty way of doing things.

If we can't, I would extract the `id` into a const at the top of the file so that you don't have it hard coded in 2 different places.